.github/workflows/fleximod_test.yaml

```diff
@@ -0,0 +1,22 @@
+on:
+  pull_request:
+    branches: [ development ]
+jobs:
+  fleximod-test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        # oldest supported and latest supported
+        python-version: ["3.7", "3.x"]
+    steps:
+      - id: checkout-CESM
+        uses: actions/checkout@v4
+      - id: run-fleximod
+        run: |
+          $GITHUB_WORKSPACE/bin/git-fleximod update
+          $GITHUB_WORKSPACE/bin/git-fleximod test
+#      - name: Setup tmate session
+#        if: ${{ failure() }}
+#        uses: mxschmitt/action-tmate@v3
```

Comment:
Can these be deleted?  I think we can be copy them from the tmate repo later if needed (https://github.com/mxschmitt/action-tmate?tab=readme-ov-file#getting-started).

Reply:
@mwaxmonsky - this is from the official code that Jim released, and is ported to all the components.  I would suggest that we keep it as it is on all the other platforms (since it doesn't sound like it does any harm)

Reply:
comments removed because the workflow has diverged anyway